roles/provision_virtual_esxi/tasks/deploy_new_esxi_hosts.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: Deploy ESXi Hosts
+  ansible.builtin.include_role:
+    name: cloud.vmware_ops.provision_vm
+  vars:
+    provision_vm_hostname: "{{ provision_virtual_esxi_hostname }}"
+    provision_vm_username: "{{ provision_virtual_esxi_username }}"
+    provision_vm_password: "{{ provision_virtual_esxi_password }}"
+    provision_vm_validate_certs: "{{ provision_virtual_esxi_validate_certs | default(omit) }}"
+    provision_vm_cluster: "{{ provision_virtual_esxi_cluster | default(omit) }}"
+    provision_vm_datacenter: "{{ provision_virtual_esxi_datacenter | default(omit) }}"
+    provision_vm_folder: "{{ provision_virtual_esxi_folder | default(omit) }}"
+    provision_vm_resource_pool: "{{ provision_virtual_esxi_resource_pool | default(omit) }}"
+    provision_vm_name: "{{ _result._esxi_name }}"
+    provision_vm_guest_id: vmkernel8Guest
+    provision_vm_state: present
```

Comment:
Who should remove those VMs? 
I think we add a playbook for that as well

Reply:
I'm saying it because you used the state `preset` as an hardcoded 
I think you should add role for removing esxi hosts that include deleting VM

you can use the role of provisioning_VM but not necessary

Reply:
ive added a new role, `manage_vcenter_host_connection`, which allows a user to add, remove, connect, or disconnect the ESXi host from vcenter. This role will not delete the vm and can be used with virtual or bare metal hosts.

The original role, `provision_virtual_esxi`, is unchaged. It will provision VMs and install the ESXi operating system from an ISO on them